SQL/database_changelog.txt

```diff
@@ -1,15 +1,25 @@
 Any time you make a change to the schema files, remember to increment the database schema version. Generally increment the minor number, major should be reserved for significant changes to the schema. Both values go up to 255.
 
-The latest database version is 6.1; The query to update the schema revision table is:
+The latest database version is 6.2; The query to update the schema revision table is:
 
-INSERT INTO `schema_revision` (`major`, `minor`) VALUES (6, 1);
+INSERT INTO `schema_revision` (`major`, `minor`) VALUES (6, 2);
 or
-INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (6, 1);
+INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (6, 2);
 
 In any query remember to add a prefix to the table names if you use one.
 
 -----------------------------------------------------
 
+Version 6.2 - 16 June 2023 by itsmeow
+Add per-character role preferences.
+
+ALTER TABLE `SS13_characters` ADD COLUMN IF NOT EXISTS `role_preferences` MEDIUMTEXT NOT NULL COLLATE 'utf8mb4_general_ci' AFTER `equipped_gear`;
```

Comment:
does DEFAULT `[]` not work here?

Reply:
See my previous response

Reply:
Got it, resolving without changes needed